Backend/controllers/auth.js

```diff
@@ -113,6 +112,15 @@ exports.getMe = async (req, res, next) => {
   });
 };
 
+// implement get all user
+exports.getAllUser = async (req, res, next) => {
+  const user = await User.find();
```

Comment:
this would return the **ALL of the user field**  please exclude some of sensitive field like `password`

Reply:
Actually, it won't return the password because in the UserSchema, the 'select' option for the password field is set to 'false' na krub eiei.

Reply:
you're right 🤓